kubernetes/istio-auth.yaml

```diff
@@ -0,0 +1,343 @@
+# GENERATED FILE. MUTUAL_TLS, RBAC and auth enabled - requires 1.6
+# TO UPDATE, modify files in istio-install and run updateVersion.sh
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: istio-ca
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        istio: istio-ca
+    spec:
+      containers:
```

Comment:
Agreed - but I was hoping separate PR would solve it.
I'm going to stop generating the istio-auth for this PR, to keep the scope narrower and avoid more
confusion.